services/pom.xml

```diff
@@ -27,6 +27,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Thanks! I didn't know that, I guess it is because I was reading some old documentation.